README.md

```diff
@@ -1,2 +1,164 @@
-# BooksApp
-A interactive library
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Books App</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Books Apps] <a name="about-project"></a>
+
+**Books App** is a project to highlight the use of wireframes.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/TR/2011/WD-html5-20110405/">HTML5</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/specs.en.html">CSS</a></li>
+    <li><a href="https://www.ecma-international.org/publications-and-standards/standards/ecma-262/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Quick and responsive UX**
+- **No repetion in code use**
+- **High performance**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+You can find a live demo in [HERE](https://lordksix.github.io/booksApp/).
```

Comment:
The link to your live demo is currently broken🥲, Kindly ensure your project is deployed or alternatively you can take away this section since we aren't required to deploy our project in this milestone👍 A broken link can lead to a frustrating user experience, and give your project a less professional look👍

<img width="736" alt="image" src="https://user-images.githubusercontent.com/64643995/233967145-c2a682cf-31f2-4641-b816-91b90aea49f7.png">